tests/phpunit/tests/SVG.php

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Tests;
+
+/**
+ * @coversDefaultClass \Google\Web_Stories\SVG
+ */
+class SVG extends \WP_UnitTestCase {
```

Comment:
There's no test for the `wp_handle_upload()` and `wp_check_filetype_and_ext()` methods

Reply:
Tests added https://github.com/google/web-stories-wp/pull/5103/commits/7995a1ea92f043fb4c3103b7dcf8ec5aaffb291a